client/src/components/Button2.js

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+const Button2 = styled.button`
```

Comment:
`Button2` is not a good name. Actually, it looks very similar to `Button`. 
Maybe you can combine these and add a property to apply different styles?